### PR TITLE
[MOSIP-13847] deprecated biometricReferenceId

### PR DESCRIPTION
**Pull request by manojsp12**

### CODE REVIEW

File: id-repository/pom.xml [hunk 1]
```diff
@@ -60,7 +60,7 @@
 
 	<modules>
 		<module>id-repository-core</module>
-		<module>id-repository-vid-service</module>
+<!-- 		<module>id-repository-vid-service</module> -->
```
Comment (LoganathanSekar7627) [inline id-repository/pom.xml:63]: Please uncomment the vid service

**Review by LoganathanSekar7627 — CHANGES_REQUESTED** [review]

File: id-repository/pom.xml (hunk 1 above)
Comment (manojsp12) [reply id-repository/pom.xml:63]: there is no changes in vid service Loga

**Review by LoganathanSekar7627 — APPROVED** [review]

**Comment by LoganathanSekar7627** [comment]
Approved

File: id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/dto/RequestDTO.java [hunk 2]
```diff
@@ -15,5 +15,6 @@
 
 	private String registrationId;
 	
+	@Deprecated(since = "1.1.4")
```
Comment (gsasikumar) [inline id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/dto/RequestDTO.java:18]: it should be since 1.1.4.3

**Review by gsasikumar — CHANGES_REQUESTED** [review]

File: id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/dto/RequestDTO.java (hunk 2 above)
Comment (manojsp12) [reply id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/dto/RequestDTO.java]: Updated to 1.1.4.3

**Review by gsasikumar — APPROVED** [review]

**Pull request merged**